specifyweb/frontend/js_src/lib/components/HomePage/TaxonTiles.tsx

```diff
@@ -30,17 +31,37 @@ export function TaxonTiles(): JSX.Element {
       treeData === undefined
     )
       return undefined;
+
+    const resizeObserver = new ResizeObserver(_.debounce(render, 50));
+
+    if (container !== null) resizeObserver.observe(container);
+
     const genusId = typeof genusRankId === 'number' ? genusRankId : undefined;
     const titleGenerator = getTitleGenerator(genusId);
-    const chart = makeTreeMap(container, treeData.root);
-    chart
-      .attr('title', titleGenerator)
-      .on('mouseover', (_event, node) => setTitle(titleGenerator(node)))
-      .on('click', (_event, node) =>
-        window.open(`/specify/query/fromtree/taxon/${node.data.id}/`, '_blank')
-      );
+
+    let chart: ReturnType<typeof makeTreeMap> | undefined = undefined;
+
+    function render(): void {
+      if (container === null || treeData === undefined) return;
+      void chart?.remove();
```

Comment:
(optional) I don't think this would be an issue, but could you double-check that there is no error if this line is executed twice? just in case

i.,e `void chart?.remove(); void chart?.remove();`